.github/workflows/release.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Nu
         uses: hustcer/setup-nu@main
         with:
-          version: '0.63.0'
-      - name: Update json
+          version: '0.64.0'
```

Comment:
don't forget to make it up to date before merge

Reply:
Done